documentation/installation.md

```diff
@@ -225,3 +224,102 @@ Debug
 To access the console, you can use `shell_plus`:
 
     make shell
+
+## OCIM Frontend
+
+The frontend of OCIM is a single-page app using the React and Redux. All the code can be found inside `frontend` directory.
+
+### Provisioning
```

Comment:
Can you move these steps to `frontend/README.md` and link the document from here?
This will keep the documentation related to the frontend in a single place.